packages/components/src/components/accordion/accordion.jsx

```diff
@@ -22,7 +27,7 @@ const Accordion = ({ className, icon_close, icon_open, list }) => {
                 <div
                     className={classNames(
                         'dc-accordion__item',
-                        `dc-accordion__item--${open_idx === idx ? 'open' : 'close'}`,
+                        `dc-accordion__item--${expand_all || open_idx === idx ? 'open' : 'close'}`,
```

Comment:
```suggestion
                        `dc-accordion__item--${is_expand_all || open_idx === idx ? 'open' : 'close'}`,
```

Reply:
Will remove state and use the prop directly